runs/comparisons/suprb_tuning.py

```diff
@@ -83,91 +89,54 @@ def suprb_ES_GA_space(trial: Trial, params: Bunch):
-        params.solution_composition__selection = getattr(
-            ga.selection, params.solution_composition__selection)()
-
-        if isinstance(params.solution_composition__selection, ga.selection.Tournament):
-            params.solution_composition__selection__k = trial.suggest_int(
-                'solution_composition__selection__k', 3, 10)
-
-        params.solution_composition__crossover = trial.suggest_categorical('solution_composition__crossover',
-                                                                           ['NPoint', 'Uniform'])
-        params.solution_composition__crossover = getattr(
-            ga.crossover, params.solution_composition__crossover)()
+        params.solution_composition__selection__k = trial.suggest_int('solution_composition__selection__k', 3, 10)
+
+        params.solution_composition__crossover = trial.suggest_categorical(
+            'solution_composition__crossover', ['NPoint', 'Uniform'])
+        params.solution_composition__crossover = getattr(ga.crossover, params.solution_composition__crossover)()
 
         if isinstance(params.solution_composition__crossover, ga.crossover.NPoint):
-            params.solution_composition__crossover__n = trial.suggest_int(
-                'solution_composition__crossover__n', 1, 10)
+            params.solution_composition__crossover__n = trial.suggest_int('solution_composition__crossover__n', 1, 10)
 
         params.solution_composition__mutation__mutation_rate = trial.suggest_float(
             'solution_composition__mutation_rate', 0, 0.1)
 
-        params.solution_composition__init__mixing__experience_weight = trial.suggest_float(
-            'solution_composition__init__mixing__experience_weight', 0, 2)
-        # params.solution_composition__init__mixing__experience_calculation__lower_bound = trial.suggest_float(
-        #     'solution_composition__init__mixing__experience_calculation__lower_bound', 0, 10)
-        params.solution_composition__init__mixing__experience_calculation__upper_bound = trial.suggest_float(
-            'solution_composition__init__mixing__experience_calculation__upper_bound', 20, 50)
-
-        params.solution_composition__init__mixing__filter_subpopulation = trial.suggest_categorical(
-            'solution_composition__init__mixing__filter_subpopulation',
-            ['FilterSubpopulation', 'NBestFitness', 'NRandom', 'RouletteWheel'])
-        params.solution_composition__init__mixing__filter_subpopulation = getattr(
-            mixing_model, params.solution_composition__init__mixing__filter_subpopulation)()
-
-        params.solution_composition__init__mixing__experience_calculation = trial.suggest_categorical(
-            'solution_composition__init__mixing__experience_calculation',
-            ['ExperienceCalculation', 'CapExperience', 'CapExperienceWithDimensionality'])
+        # Mixing
+        params.solution_composition__init__mixing__filter_subpopulation__rule_amount = rule_amount
```

Comment:
a little surprised this works but even better

Reply:
It's basically setting the param in the same way as a trial would, but only that 1 value